bug: record_history()에 source_id/work_id 미설정 — 빌드 실패

## 문제

PR #36에서 `HistoryEntry`에 `source_id`, `work_id` 필드가 추가되었으나, `Daemon::record_history()`가 업데이트되지 않아 **컴파일 실패**.

```
error[E0063]: missing fields `source_id` and `work_id` in initializer of `HistoryEntry`
  --> crates/belt-daemon/src/daemon.rs:431
```

## 수정

`daemon.rs:431`의 `HistoryEntry` 생성에 추가:
```rust
source_id: item.source_id.clone(),
work_id: item.work_id.clone(),
```

## Severity
Critical — 프로젝트 빌드 불가

--- a/crates/belt-daemon/src/daemon.rs
+++ b/crates/belt-daemon/src/daemon.rs
@@ -441,6 +441,8 @@ impl Daemon {
             .count() as u32
             + 1;
         self.history.push(HistoryEntry {
+            source_id: item.source_id.clone(),
+            work_id: item.work_id.clone(),
             state: item.state.clone(),
             status: status
                 .parse()

